openrndr-core/src/main/kotlin/org/openrndr/shape/Segment.kt

```diff
@@ -677,9 +681,10 @@ class Segment {
         }
 
     /**
-     * Take a sub segment starting at [startT] and ending at [endT]
-     * @param startT starting segment t parameterization
-     * @param endT starting segment t parameterization
+     * Samples a [Segment] starting at [startT] and ending at [endT].
```

Comment:
Another puzzling word for me, because technically "sampling" something means taking a subset of something by some predicate, which makes sense to me in this context. So whenever the word "sample" is used in the docs, it is that meaning in particular. The word was used elsewhere in the docs already to refer to similar functions, so I thought I'd use it here for the sake of consistency. Should I replace all instances of it with something more plain?

Reply:
I guess I'm influenced by sampling in music. A sample can be both a short sound clip but then we also talk about samples per second where one sample is be just a float value. This second use is what comes first to mind. But I can rewire my brain :)

Reply:
But think of it this way, as you said the sample can be a short sound clip, but is it not a short clip from a bigger piece?

Reply:
Ok, for me it works and I know what it does. There's always time for adjustments :-)